src/components/charts/TempChart.vue

```diff
@@ -312,8 +312,11 @@ export default class TempChart extends Mixins(BaseMixin) {
             output +=
                 '<div class="row">' +
                 '<div class="col py-1" style=\'border-bottom: 1px solid rgba(255, 255, 255, 0.24);\'>' +
-                "<span class='v-icon mdi theme-dark' style='font-size: 14px; margin-right: 5px;'>" +
-                mdiClock +
+                "<span class='v-icon mdi theme-dark' style='margin-right: 5px;'>" +
```

Comment:
this line have the quotes reverse. first the double quotes and in the string the single quotes. can you please adjust this?

Reply:
Will do